hr_employee_document/views/hr_employee_views.xml

```diff
@@ -11,7 +11,6 @@
         <field name="inherit_id" ref="hr.view_employee_form"/>
         <field name="groups_id" eval="[(6, 0, [ref('hr.group_hr_user') ])]"/>
         <field name="arch" type="xml">
-            <field name="google_drive_link" position="replace"/>
```

Comment:
This is part of migration or improvement?

Reply:
Part of migration

Reply:
Yes, Migration

Reply:
That field is gone from Odoo?

Reply:
@alexey-pelykh I can not see `google_drive_link` field in employee view.
![Selection_459](https://user-images.githubusercontent.com/24691983/97609749-9123af80-1a46-11eb-9886-42b6b87f3dfd.png)